sdk/communication/azure-communication-phonenumbers/test/sip_routing_helper.py

```diff
@@ -0,0 +1,59 @@
+# ------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+
+import os
+
+from devtools_testutils import is_live
+
+from azure_devtools.scenario_tests import RecordingProcessor
+
+def get_user_domain():
+    if(is_live()):
+        sip_domain = os.getenv("AZURE_TEST_SIP_DOMAIN")
+        assert sip_domain is not None, "Missing AZURE_TEST_SIP_DOMAIN environment variable."
+        return sip_domain
+    return "sanitized.com"
+
+def trunks_are_equal(response_trunks, request_trunks):
```

Comment:
When I read a function called `trunks_are_equal()` I expect to get a Boolean result indicating whether the objects are equal or not, instead of asserting something, which will break the code execution. 

To be more pythonic, you can implement the `__eq__` method in the Trunk class and then assert the equality in the test itself.

If implementing such method is not an option, I think this helper should be named in a way that makes it clear that it's intended to be used in tests only, and also make it clear that this method asserts equality, instead of just returning a Boolean value.

Reply:
Changed naming to assert_trunks_are_equal and assert_routes_are_equal for better clarity.